### PR TITLE
Bugfix/board style

### PR DESCRIPTION
**Pull request by DiamondThree**

### CODE REVIEW

File: libs/components/affine-board/src/Board.tsx [hunk 1]
```diff
@@ -51,11 +51,10 @@ const AffineBoard = ({
         };
     });
 
-    const { shapes, bindings } = useShapes(workspace, rootBlockId);
-
+    const { shapes } = useShapes(workspace, rootBlockId);
     useEffect(() => {
         if (app) {
-            app.replacePageContent(shapes || {}, bindings, {});
+            app.replacePageContent(shapes || {}, {}, {});
```
Comment (alt1o) [inline libs/components/affine-board/src/Board.tsx:57]: Maybe you should remove the save bindings logic

File: libs/components/affine-board/src/hooks/use-shapes.ts [hunk 2]
```diff
@@ -43,11 +43,9 @@ export const useShapes = (workspace: string, rootBlockId: string) => {
                     return shapes;
                 }),
         ]).then(shapes => {
-            getBindings(workspace, rootBlockId).then(bindings => {
-                setBlocks({
-                    shapes,
-                    bindings: bindings,
-                });
+            setBlocks({
```
Comment (alt1o) [inline libs/components/affine-board/src/hooks/use-shapes.ts:46]: Maybe you should remove the 'getBindings' implement.

File: libs/components/board-draw/src/TlDraw.tsx [hunk 3]
```diff
@@ -361,11 +361,11 @@ const InnerTldraw = memo(function InnerTldraw({
...
-    const hideBounds =
-        (isInSession && app.session?.constructor.name !== 'BrushSession') ||
-        !isSelecting ||
-        isHideBoundsShape ||
-        !!pageState.editingId;
+    // const hideBounds =
+    //     (isInSession && app.session?.constructor.name !== 'BrushSession') ||
+    //     !isSelecting ||
+    //     isHideBoundsShape ||
+    //     !!pageState.editingId;
```
Comment (alt1o) [inline libs/components/board-draw/src/TlDraw.tsx:368]: Should the selected bounds be displayed in all cases? For example, during brush painting

**Review by alt1o — APPROVED** [review]

File: libs/components/board-draw/src/TlDraw.tsx (hunk 3 above)
Comment (DiamondThree) [reply libs/components/board-draw/src/TlDraw.tsx]: NO, i will fix the bug

**Pull request merged**